build/tasks/test.js

```diff
@@ -59,7 +58,7 @@ gulp.task('enforce-code-coverage', ['run-unit-tests'], function() {
     ));
 });
 
-gulp.task('show-unit-test-coverage-report', ['run-unit-tests'], function() {
+gulp.task('show-unittest-coverage-report', ['run-unit-tests'], function() {
```

Comment:
Thank you for making this consistent with the npm script name :) can't tell you how many times I have mixed the two up